css/style.css

```diff
@@ -0,0 +1,182 @@
+:root {
+    --basic-black: #000000;
+    --basic-white: #ffffff;
+    --basic-blue: #23A9E1;
+    --basic-grey: #333333;
+    --basic-brown: #766357;
+    --basic-blue1: #81B3D2;  
+    --basic-white1: #F2F2F2;
+    --basic-grey2: #A9A9A9;
+    --basic-blue2: #81B3D2;
+    --basic-white2: #eeeeee;
+}
+
+.page-body {
+  min-width: 960px;
+  margin: 0;
+  padding: 0;
+  font-family: "PT Sans", Arial, sans-serif;
+  font-size: 21px;
+  line-height: 26px;
+  font-weight: 700px;
+  color: var(--basic-black);
+  text-transform: uppercase;
+  background-color: var(--basic-white2);
+  background-image: url("img/background.jpg");
```

Comment:
Это изображение не нужно применять ко всей странице. Его нужно использовать в качестве фона в этом блоке:
![image](https://user-images.githubusercontent.com/17790771/123512538-ab269800-d690-11eb-9730-9db684dbd8dc.png)
Потребуется обернуть изображение welcome.svg в `div` и фон background.jpg применить к этому `div`.

Когда добавляете фоновые изображения в CSS, то в пути к файлу нужно добавить эти символы `../` То есть должно быть так:
```suggestion
  background-image: url("../img/background.jpg");
```
Из-за их отсутствия браузер не находит изображение. Про относительные и абсолютные пути подробнее в [учебнике](https://up.htmlacademy.ru/htmlcss/31/book/2.3).